### PR TITLE
Add PHP Swoole

### PR DESCRIPTION
**Pull request by Alban-io**

### CODE REVIEW

File: php_swoole_grpc_bench/server.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+]);
+
+$http->on('request', function ($request, $response) {
+
+    try {
+        $request_message = new HelloRequest;
+        $request_message->mergeFromString(Parser::unpack($request->rawContent()));
+
+        $response_message = new HelloReply();
+        $response_message->setMessage("Hello {$request_message->getName()}");
```
Comment (LesnyRumcajs) [inline php_swoole_grpc_bench/server.php:24]: You don't need to concatenate with "Hello", in other benchmarks we just set the same message as received. Perhaps it should've been called "EchoRequest" but it's kind of too late for that. :)

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]

File: php_swoole_grpc_bench/server.php (hunk 1 above)
Comment (Alban-io) [reply php_swoole_grpc_bench/server.php]: Ok, I fixed that :)

**Review by LesnyRumcajs — APPROVED** [review]

File: php_swoole_grpc_bench/server.php (hunk 1 above)
Comment (LesnyRumcajs) [reply php_swoole_grpc_bench/server.php]: Thank you!

**Comment by LesnyRumcajs** [comment]
Looks good, thanks for the contribution!

**Comment by Alban-io** [comment]
I'm happy to contribute 🙂

**Pull request merged**